### PR TITLE
removed bpp_base and changed the n_batches_per_pass to 1 by default

### PR DESCRIPTION
**Pull request by BordiaS**
Modified `main.py` and  train function in `src/trainer.py`

### CODE REVIEW

**Comment by sleepinyourhat** [comment]
Reviewing

**Comment by sleepinyourhat** [comment]
Though, @pruksmhc, mind taking a quick look to make sure there's no subtle behavior in trainer that depends on bpp_base?

**Comment by sleepinyourhat** [comment]
One more request, @BordiaS: Make sure to delete bpp_base from _all_ checked-in config files. It's still there in a few places.

**Comment by BordiaS** [comment]
> Looks good. Please run demo.conf to completion to make sure it works, then let me know and I'll merge.
Had a minor issue. `demo.conf` now runs as it was running earlier.

**Comment by BordiaS** [comment]
> One more request, @BordiaS: Make sure to delete bpp_base from _all_ checked-in config files. It's still there in a few places.
Removed.

File: config/demo.conf [hunk 1]
```diff
@@ -14,7 +14,13 @@
 include "defaults.conf"  // relative path to this file
 
 // write to local storage by default for this demo
-project_dir = ${JIANT_PROJECT_PREFIX}
```
Comment (sleepinyourhat) [inline config/demo.conf]: Undo this first set of changes—not related to this PR.

Comment (BordiaS) [reply config/demo.conf]: Fixed it.

File: src/trainer.py [hunk 2]
```diff
@@ -532,8 +532,10 @@ def clip_function(
             scheduler = g_scheduler if shared_optimizer else task_info['scheduler']
             total_batches_trained = task_info['total_batches_trained']
             n_batches_since_val = task_info['n_batches_since_val']
-            tr_loss = task_info['loss']
-            for batch in itertools.islice(tr_generator, n_batches_per_pass):
+            tr_loss = task_info['loss'];
+
```
Comment (sleepinyourhat) [inline src/trainer.py]: Don't use more than one line break inside a function.

File: src/trainer.py [hunk 3]
```diff
@@ -532,8 +532,10 @@ def clip_function(
             scheduler = g_scheduler if shared_optimizer else task_info['scheduler']
             total_batches_trained = task_info['total_batches_trained']
             n_batches_since_val = task_info['n_batches_since_val']
-            tr_loss = task_info['loss']
-            for batch in itertools.islice(tr_generator, n_batches_per_pass):
+            tr_loss = task_info['loss'];
```
Comment (sleepinyourhat) [inline src/trainer.py]: Is this valid syntax? Looks very unusual.

File: config/demo.conf [hunk 4]
```diff
@@ -14,7 +14,13 @@
 include "defaults.conf"  // relative path to this file
 
 // write to local storage by default for this demo
+
+
 project_dir = ${JIANT_PROJECT_PREFIX}
+
+word_embs_file =""
```
Comment (sleepinyourhat) [inline config/demo.conf]: Again, revert all these changes. You should only change lines that are related to this issue.

**Comment by pruksmhc** [comment]
This looks good!

**Pull request merged**